### PR TITLE
Mes 3664 pass debrief restructure folders for cat be

### PR DESCRIPTION
**Pull request by appleabs**
## Description
Refractor folder structure for pass debrief page for category be.
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/pass-finalisation/cat-b/pass-finalisation.module.cat-b.page.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
```
Comment (TomBillingtonUK) [inline src/pages/pass-finalisation/cat-b/pass-finalisation.module.cat-b.page.ts]: This page should follow this file name pattern

waiting-room.cat-b.page.module

**Comment by TomBillingtonUK** [comment]
You don't seem to have changed Page.constants file with the new name so I don't think this page currently loads correctly

**Review by TomBillingtonUK — APPROVED** [review]
Looks good , a couple of comments but nothing major :)

**Review by MarkPatt89 — APPROVED** [review]
Sorry for the delay

**Pull request merged**